app.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peeps'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    redirect '/peeps'
```

Comment:
This page seems only to redirect to another page.. maybe the peeps could be moved here? Just a thought!